tests/rt.conf

```diff
@@ -25,14 +25,14 @@ COMPILE | s2swa_32bit  | intel | -DAPP=S2SWA -D32BIT=ON -DCCPP_SUITES=FV3_GFS_v1
 RUN | cpld_control_p8_mixedmode                         |    - noaacloud                     | baseline |
 
 #GFS
-COMPILE | s2swa_32bit_pdlib  | intel | -DAPP=S2SWA -D32BIT=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8_ugwpv1 -DPDLIB=ON | + hera orion cheyenne wcoss2 acorn | fv3 |
-RUN | cpld_control_gfsv17                               | + hera orion cheyenne wcoss2 acorn | baseline |
-RUN | cpld_control_gfsv17_iau                           | + hera orion cheyenne wcoss2 acorn | baseline | cpld_control_gfsv17
-RUN | cpld_restart_gfsv17                               | + hera orion cheyenne wcoss2 acorn |          | cpld_control_gfsv17 
-RUN | cpld_mpi_gfsv17                                   | + hera orion cheyenne wcoss2 acorn |          |
+COMPILE | s2swa_32bit_pdlib  | intel | -DAPP=S2SWA -D32BIT=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8_ugwpv1 -DPDLIB=ON | - noaacloud | fv3 |
+RUN | cpld_control_gfsv17                               |    - noaacloud                     | baseline |
+RUN | cpld_control_gfsv17_iau                           |    - noaacloud                     | cpld_control_gfsv17
```

Comment:
@jkbk2004 I have been testing this with scotch 7.0.4 on wcoss2 and discovered there's an issue here.  I unintentionally deleted the `| baseline` which was somehow not caught in my testing on gaea.      

I'm in the process of re-running tests now and will re-run tests on hera as well with this new change.

Reply:
Sure! We may go the NST bug fix PR first today. We can pick up this PR Monday. Let me know.

Reply:
NST bug fix PR is #2001